getting-started/templates/systemlink-values.yaml

```diff
@@ -617,27 +617,51 @@ sl-jupyterhub:
     annotations:
       nginx.ingress.kubernetes.io/proxy-body-size: 100m
 
-## Configuration for Notebook Execution workers
+## Configuration for Argo Workflows.
 ##
-nbexecworker:
-  ## Customize routes to back-end APIs
-  ##
-  apiRoutes:
-    ## Sets the release name prefix for the Jupyter Enterprise Gateway service running on the cluster.
-    # <ATTENTION> This must be set to the release name used to install the ni-helm/systemlinkadmin chart.
-    ##
-    jupyterEnterpriseGatewayReleaseName: "systemlink-admin"
-    ## Sets the service name for the Jupyter Enterprise Gateway instance running on the cluster.
-    ##
-    jupyterEnterpriseGatewayServiceName: "enterprise-gateway"
-    ## Sets the namespace name for the Jupyter Enterprise Gateway service running on the cluster.
-    # <ATTENTION> This must be set to the namespace where the ni-helm/systemlinkadmin chart was installed.
-    ##
-    jupyterEnterpriseGatewayNamespace: "systemlink-admin"
-    ## Sets the port number of for the Jupyter Enterprise Gateway service running on the cluster.
-    ##
-    jupyterEnterpriseGatewayPort: "8888"
-    ## If set, overrides the other jupyterEnterpriseGateway options and configures the service URL directly.
+argoworkflows:
+  argo-workflows:
+    images:
+      pullSecrets: *pullSecrets
+    controller:
+      image:
+        registry: *imageRegistryRef
+    executor:
+      image:
+        registry: *imageRegistryRef
+    server:
+      image:
+        registry: *imageRegistryRef
+
+## Configuration for Notebook Execution service.
+##
+nbexecservice:
+  argo:
+    ## Configure S3/MinIO access.
     ##
-    jupyterEnterpriseGatewayUrl: ""
-
+    artifactRepository:
+      s3:
+        ## The name of the S3 or MinIO bucket that the service should connect to.
+        ##
+        bucket: "systemlink-notebook-execution"
+        ## Set this value to connect to an external S3 instance.
+        # <ATTENTION> To connect to an external S3 bucket, set the host here as well as the port.
+        ##
+        host: ""
+        ## Set this value to connect to an S3 instance which is internal to the cluster. Ignored if host is set.
+        ##
+        service: *minioServiceName
```

Comment:
This should have an <ATTENTION> block and instructions for what a user should change if they are not using minio and are instead using external S3 storage. GM will need to setup the latter.

Reply:
Not sure I'm 100% understanding this.

This is following the same pattern as file ingestion and data frame, both using s3. There are some instructions to the host option (in case of external s3 service): "# <ATTENTION> To connect to an external S3 bucket, set the host here as well as the insecure and port."

Reply:
I think I missed your instructions on my read through. This is fine.